cloud/aws/runtime/core/provider.go

```diff
@@ -66,6 +66,10 @@ type awsProviderImpl struct {
 
 var _ AwsProvider = &awsProviderImpl{}
 
+func (a *awsProviderImpl) Declare(ctx context.Context, typ common.ResourceType, name string) error {
+	return nil
+}
```

Comment:
Might also want to include additional resource details from the payload to save us have to re-implement this later if we're missing information.